### ISSUE TITLE
Touchable mocks should respect `disabled` props

### ISSUE THREAD

**Issue by sbycrosz** [lelandrichardson/react-native-mock#73]
Should we have `Touchable` mocks to respect `disabled` props (i.e. not to call `onPress` callback when it has `disabled` props)? or is this outside the scope of this library?

**Comment by RealOrangeOne** [comment]
No, that's certainly something for this library. If you would like to open a PR on this it would be great!

**Comment by sbycrosz** [comment]
Alrighty, will do a PR sometime later
